.circleci/config.yml

```diff
@@ -478,6 +485,9 @@ workflows:
       - just_simulate_ink_sep_holocene_system_config_upgrade:
           context:
             - circleci-repo-readonly-authenticated-github-token
+      - just_simulate_op_sep_prestate_update:
```

Comment:
let's work with @Ethnical @sbvegan to use the `simulate_sequence` here, since we have 3 ongoing sepolia tasks to sequence. They are easy to resign, but still good practice as prep for the inevitable mainnet sequencing

Reply:
I'd like to execute ink-001 tomorrow morning. Can we use `simulate_sequence` for this task and ink-002?

Reply:
001ddf9a4fa3